include/RAJA/exec-agency/forallN_agency.hxx

```diff
@@ -119,7 +119,7 @@ struct ForallN_Executor<ForallN_PolicyPair<agency_base<Agent, Worker>,
         agency::experimental::interval(begin_i, end_i), numThreads);
 
     agency::bulk_invoke(Worker{}(tiles.size()),
-                        [=](Agent& self) {
+                        [&](Agent& self) {
```

Comment:
Duplicate of the changes in #7 (although I didn't do forallN in that one) but necessary to make things work.